.gitlab-ci.yml

```diff
@@ -20,7 +20,7 @@ variables:
   LINUX_FORTRAN_COMPONENTS_WEB: intel.oneapi.lin.ifort-compiler
   LINUX_DPCPP_COMPONENTS_WEB: intel.oneapi.lin.dpcpp-cpp-compiler
   LINUX_APT_DNF_CACHE_NUMBER: 2
```

Comment:
@daniil-mazur, please update the cache number to 3, so that the CI cache is updated after merge.

Reply:
@mmzakhar fixed